apps/src/p5lab/spritelab/TextConsole.jsx

```diff
@@ -76,30 +77,28 @@ export default class TextConsole extends React.Component {
 
   timeout = 0;
 
-  toggleConsole() {
-    this.state.open ? this.closeConsole() : this.openThenClose();
+  toggleConsole(autoCloseTime) {
```

Comment:
[applies to both this method and the `openConsole` method] i'll usually add a default value for any optional parameters like `autoCloseTime` to communicate that the param is optional, which i think would be useful here. another option would be to add a comment with that info -- i'd be okay with either of those resolutions